.swiftlint.yml

```diff
@@ -1,55 +1,49 @@
+# Find all the available rules by running:
+# swiftlint rules
+# https://realm.github.io/SwiftLint/ to look up what they do
+
 disabled_rules: # rule identifiers to exclude from running
-  - force_cast
   - line_length
-#  - function_body_length
-#  - trailing_newline
-#  - opening_brace
-#  - comma
-#  - colon
-#  - type_name
-#  - variable_name_min_length
-#  - trailing_semicolon
-#  - force_try
-#  - nesting
-#  - variable_name
-#  - conditional_binding_cascade
-#  - variable_name_max_length
-#  - operator_whitespace
-#  - control_statement
-#  - legacy_constant
-#  - line_length
-#  - return_arrow_whitespace
-#  - trailing_whitespace
-#  - closing_brace
-#  - statement_position
-#  - type_body_length
-#  - file_length
-#  - leading_whitespace
-#  - legacy_constructor
-#  - todo
-#  - valid_docs
-#  - missing_docs
-#  - trailing_newline
 
 opt_in_rules: # some rules are only opt-in
-  - empty_count # Prefer checking isEmpty over comparing count to zero
+  - empty_collection_literal # Prefer checking isEmpty over comparing to an empty collection.
+  - empty_count # Prefer checking isEmpty over comparing count to zero.
   - empty_string # Prefer checking isEmpty over comparing string to an empty string literal.
-  - unused_private_declaration # prevents unused private declarations
-  # Find all the available rules by running:
-  # swiftlint rules
+  - expiring_todo # TODOs and FIXMEs should be resolved prior to their expiry date.
```

Comment:
What is the deadline or expiry date then?

Reply:
i think you add them on each todoo